src/test/config/bitbucket/values-KITT.yaml

```diff
@@ -21,7 +21,7 @@ volumes:
   sharedHome:
     customVolume:
       persistentVolumeClaim:
-        claimName: efs-claim # Pre-provisioned, and shared by all of our pods
+        claimName: ${shared.pvc.name} # Pre-provisioned, and shared by all of our pods
```

Comment:
Hmm, maybe we should prefix it with a namespace instead? One less property to worry about. Something like
`${namespace}-atlassian-dc-shared-home`